install/kubernetes/cilium/Chart.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: cilium
 displayName: Cilium
 home: https://cilium.io/
-version: 1.15.0-dev
-appVersion: 1.15.0-dev
-kubeVersion: ">= 1.16.0-0"
-icon: https://cdn.jsdelivr.net/gh/cilium/cilium@main/Documentation/images/logo-solo.svg
+version: 1.15.0-pre.3
+appVersion: 1.15.0-pre.3
+kubeVersion: ">= 1.26.0-0"
```

Comment:
We haven't disclaimed anywhere that we aren't supporting k8s < 1.26. Maybe we should.

Reply:
I don't think we've made some kind of announcement. It will be part of the docs page though: https://docs.cilium.io/en/latest/network/kubernetes/compatibility/#k8scompatibility .

I guess this is feedback to think about how we communicate this more clearly? Not sure whether to go and find the relevant PR and just bump its release-note to major or do something else...?